### PR TITLE
refactor "participative_assistant" to "ludens"

### PR DESCRIPTION
**Pull request by eliegaboriau**
fix issue #1

### CODE REVIEW

**Review by paulinebessoles — APPROVED** [review]

File: lib/decidim/ludens.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "decidim/ludens/admin"
+require "decidim/ludens/engine"
+require "decidim/ludens/admin_engine"
+require "decidim/ludens/component"
+
+module Decidim
+  # This namespace holds the logic of the `ParticipativeAssistant` component. This component
```
Comment (Quentinchampenois) [inline lib/decidim/ludens.rb]: Can you update this outdated comment please ?

File: lib/decidim/ludens/admin.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Decidim
-  module ParticipativeAssistant
+  module Ludens
     # This module contains all the domain logic associated to Decidim's ParticipativeAssistant
```
Comment (Quentinchampenois) [inline lib/decidim/ludens/admin.rb]: Can you update this outdated comment please ?

File: lib/decidim/ludens/admin_engine.rb [hunk 3]
```diff
@@ -1,36 +1,36 @@
 # frozen_string_literal: true
 
 module Decidim
-  module ParticipativeAssistant
+  module Ludens
     # This is the engine that runs on the public interface of `ParticipativeAssistant`.
```
Comment (Quentinchampenois) [inline lib/decidim/ludens/admin_engine.rb]: Can you update this outdated comment please ?

File: lib/decidim/ludens/component.rb [hunk 4]
```diff
@@ -27,7 +27,7 @@
   # component.register_resource(:some_resource) do |resource|
   #   # Register a optional resource that can be references from other resources.
   #   resource.model_class_name = "Decidim::ParticipativeAssistant::SomeResource"
```
Comment (Quentinchampenois) [inline lib/decidim/ludens/component.rb]: Can you update this outdated comment please ?

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Nice refactor, I have few requests about comments documentation

**Review by Quentinchampenois — APPROVED** [review]

**Pull request merged**